provisioning/playbooks/requirements.yml

```diff
@@ -1,3 +1,4 @@
 ---
 - src: geerlingguy.mysql
 - src: geerlingguy.docker
+- src: realsalmon.mysql-client
```

Comment:
Your solution was to add an extra role: "**realsalmon.mysql-client**"
My suggestion was to define variables for an **existing** role: "**geerlingguy.mysql**"
by adding to the file provisioning\playbooks\group_vars\all.yml
the following lines:

```
mysql_packages
  - mysql-common
  - mysql-server
  - libmysqlclient-dev
  - mysql-client
```
What is the better way ?

Reply:
Because the additional package is for mysql client. The role we are using is mysql server. I would suggest to keep the two roles separate than mixing them.